lib/perl/Genome/Utility/Test.pm

```diff
@@ -312,10 +323,27 @@ sub data_dir_ok {
     return $data_dir;
 }
 
+sub get_test_dir {
```

Comment:
Good point.  Maybe I should have left `get_test_dir` in the variant reporting framework, and used `data_dir_ok` here.